### DISCUSSION TITLE
Is it possible to add undoredo action for app.setPageSize()?

### DISCUSSION THREAD

**Discussion by MiltonBalaOfficial** [xournalpp/xournalpp#6092] [Q&A] [answered]
Is it possible for this: 

-- Increase page width for the current page
`function increaseSizeRight()
  local docStructure = app.getDocumentStructure()
  local page = docStructure["currentPage"]
  local width = docStructure["pages"][page]["pageWidth"]
  local height = docStructure["pages"][page]["pageHeight"]

  app.setPageSize(width + 71, height)
end`

**Comment by bhennion** [comment, marked as answer]
It is an identified bug #6030. It should get fixed before 1.2.5 is out.
